src/main/pages/setup-docker/docker-config.adoc

```diff
@@ -108,59 +109,76 @@ CHE_JDBC_MAX__WAIT__MILLIS=-1
-== JAVA_OPTS
+== Setting the JAVA_OPTS environment variable
 
-There can be several Java processes running in a workspace machine. Some of the Java agents are special purpose agents started in a machine to provide core and additional IDE functionality. These are workspace agent and a link:dependency-management.html[Maven plugin] that are both started in own JVM. On top of that, you can run own Java programs and use build tools like Maven. A set of the following environment variables can help optimize RAM consumption:
+There can be several Java processes running in a workspace machine. Some Java agents are special purpose agents started in a machine to provide core and additional IDE functionalities. These are workspace agents and a link:dependency-management.html[Maven plugin] that are both started in the JVM. You can run your own Java programs and use build tools like Maven. A set of the following environment variables can help optimize RAM consumption:
 
-*User-Defined Envs*
+=== User-defined environment variables
 
-A user can provide own link:env-variables.html[environment variables] per workspace machine:
+You can provide your own link:env-variables.html[environment variables] per workspace machine.
 
-----
-JAVA_OPTS                                    # machine-wide java opts
-MAVEN_OPTS                                   # machine-wide maven opts
-CHE_WORKSPACE_WSAGENT__JAVA__OPTIONS           # java opts to adjust java opts of ws-agent
-CHE_WORKSPACE_MAVEN__SERVER__JAVA__OPTIONS   # java opts to adjust java opts of the maven server
-----
+JAVA_OPTS::
+machine-wide java opts
 
-Che admins (basically whoever has access to `che.env` or Che server environment directly) can override user-defined envs:
+MAVEN_OPTS::
+machine-wide maven opts
 
-----
-CHE_WORKSPACE_JAVA__OPTIONS                 # overrides the default value of JAVA_OPTS of all workspaces
-CHE_WORKSPACE_MAVEN__OPTIONS                # overrides the default value of MAVEN_OPTS of all workspaces
-CHE_WORKSPACE_WSAGENT__JAVA__OPTIONS        # overrides the default value of JAVA_OPTS of all ws-agents
-CHE_WORKSPACE_MAVEN__SERVER__JAVA__OPTIONS  # overrides the default value of JAVA_OPTS of all maven servers
+CHE_WORKSPACE_WSAGENT__JAVA__OPTIONS::           
+java opts to adjust java opts of ws-agent
+
+CHE_WORKSPACE_MAVEN__SERVER__JAVA__OPTIONS::   
+java opts to adjust java opts of the maven server
+
+Che administrators (anyone with access to the `che.env` file or the Che server environment directly) can override the following user-defined environment variables:
+
+CHE_WORKSPACE_JAVA__OPTIONS::                
+Overrides the default value of JAVA_OPTS of all workspaces
+
+CHE_WORKSPACE_MAVEN__OPTIONS::
+Overrides the default value of MAVEN_OPTS of all workspaces
+
+CHE_WORKSPACE_WSAGENT__JAVA__OPTIONS::
+Overrides the default value of JAVA_OPTS of all ws-agents
+
+CHE_WORKSPACE_MAVEN__SERVER__JAVA__OPTIONS::
+Overrides the default value of JAVA_OPTS of all maven servers
```

Comment:
Remove '----' form line 168, the rendering for lines 170 to 182 has been altered because of this

Reply:
Thanks so much, Preeti! Not sure how I missed this.